src/components/vm/disks/diskAdd.jsx

```diff
@@ -74,7 +74,7 @@ function getDiskUsageMessage(vms, storagePool, volumeName) {
 
     let message = cockpit.format(_("This volume is already used by $0."), vmsUsing);
     if (volume.format === "raw")
-        message += " " + _("Attaching it will make this disk shareable for every VM using it.");
+        message += " " + _("Adding this disk will change its mode to shared.");
```

Comment:
Thanks

- I like `s/Attach/Add` and `s/shareble/shared` , since those are the terms we use in the UI.
- Maybe we should use "access" instead of "mode" to be consistent with the term we use in the UI? ![Screenshot from 2022-12-15 14-22-49](https://user-images.githubusercontent.com/42733240/207869988-e2028da7-07d5-4dcd-9259-2812ead84bd6.png)
Or maybe combine it to "access mode" ? 
- I would still mention that the mode will be changed to share **for every VM using it**. Since it will set access mode to shared for every VM, not just the VM where the disk is being added to. The new text you suggest kinda feels like it will set the mode only for the disk added to the VM2, doesn't it? But maybe that's just my feeling.
Anyway, what about message
``suggestion
        message += " " + _("Adding this disk will change its access mode to shared or every VM using it.");
```

Reply:
Ah, I just realized that the message will always be preceded by `This volume is already used by [VM NAME]`. So probably "for every VM using it" is unnecessary, since it's implied by the context of the preceding sentence? But I'm not an English native speaker so I'm not 100% confident ¯\_(ツ)_/¯

Reply:
Yeah, I think it's implied that it's for the VMs using it (which are mentioned already).

It's not changing its "access" to shared, but it is changing its "access mode" to shared. Good call for consistency and clarity. "Adding this disk will change its access mode to shared."

You're right on both points. :+1:

Reply:
:1st_place_medal: This gets an "OK, let's merge!" from me after this change.

(If you need official approval after the change, you can check with someone else on the team within the next 2 weeks. Happy holidays!)

Reply:
The PR uses "access mode" now, so resolving this thread and assuming Garrett's approval as above.